src/box/box.cc

```diff
@@ -2213,6 +2214,8 @@ box_make_bootstrap_leader(void)
 	} else {
 		bootstrap_leader_uuid = INSTANCE_UUID;
 		is_supervised_bootstrap_leader = true;
+		say_info("instance %s (this one) is assigned as a bootstrap "
```

Comment:
No need for this message. It's duplicated, since in `bootstrap_master -> box_populate_schema_space -> box_set_bootstrap_leader_record` it'll be written to `_schema` and the message will be logged from the `on_commit` trigger

Reply:
Agree, removed.